lib/index.js

```diff
@@ -557,6 +558,10 @@ export default {
 				args.push('--liveview');
 			}
 
+			if (isDebugMode) {
+				args.push('--debug-host localhost:8989');
```

Comment:
I think this would need to be the below. Adding as a single string versus adding as two separate strings appears to change how things are passed into the spawned process and then the arg handling
 
```suggestion
				args.push('--debug-host', 'localhost:8989');
```

Reply:
nice, now its working